.flowconfig

```diff
@@ -16,3 +16,6 @@ esproposal.class_static_fields=enable
 esproposal.class_instance_fields=enable
 unsafe.enable_getters_and_setters=true
 module.name_mapper='/lib/' -> '/src/'
+
+[version]
+^0.54.0
```

Comment:
Flow gets updated pretty often. It seems to me that this will either tie us to an old version or require us to update  this file every time flow pushes a new version. Also, what is the role of yarn.lock in all this?

Reply:
@hermanventer, "yarn flow" already ties to an old version here:
https://github.com/facebook/prepack/blob/master/package.json#L81
This diff just matches the system flow version with what "yarn flow" uses.
When we are ready to update, we should update both places.

Reply:
OK, this is coming back to me now. I am not in favor of lagging behind the latest Flow in our editing environment. I rather like to be reminded to update package.json whenever a Flow type error shows up in the editor, but not in yarn flow.